scripts/validate_docstrings.py

```diff
@@ -679,5 +760,24 @@ def header(title, width=80, char='#'):
                            nargs='?',
                            default=None,
                            help=func_help)
+    argparser.add_argument('--format', default='default', help='format of the '
+                           'output when validating multiple docstrings '
+                           '(ignored when validating one). '
+                           'It can be {}'.format(str(format_opts)[1:-1]))
+    argparser.add_argument('--prefix', default=None, help='pattern for the '
+                           'docstring names, in order to decide which ones '
+                           'will be validated. A prefix "pandas.Series.str.'
+                           'will make the script validate all the docstrings'
+                           'of methods starting by this pattern. It is '
+                           'ignored if parameter function is provided')
+    argparser.add_argument('--errors', default=None, help='comma separated '
```

Comment:
The default of None is causing issues for me locally.

```
$ python scripts/validate_docstrings.py pandas.DataFrame.head
Traceback (most recent call last):
  File "scripts/validate_docstrings.py", line 782, in <module>
    sys.exit(main(args.function, args.prefix, args.errors.split(','),
AttributeError: 'NoneType' object has no attribute 'split'
```

Reply:
For single docstrings, is it ignored by choice or for simplicity?

Reply:
Oh, that's a bug I introduced at the last minute, my fault.

It's more by choice, I don't see a use case where you're validating a single docstring, and you just care about a subset of errors. It's easy to implement, but I would prefer that nobody does that (unless there is a use case I'm not considering).

Reply:
Works for me.